### PR TITLE
Fulp code improvements

### PR DESCRIPTION
**Pull request by Bilbo367**
## About The Pull Request

Fixes spelling mistakes, organizes some files, removes unused important_prison_messages, formatting, and comment changes.

## Why It's Good For The Game

Should make code more understandable.

### CODE REVIEW

File: fulp_modules/features/clothing/neck/beefman_locket.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/item/clothing/neck/bfeemam
+/obj/item/clothing/neck/beefman
```
Comment (JohnFulpWillard) [inline fulp_modules/features/clothing/neck/beefman_locket.dm:1]: This was intentional but whatever

File: fulp_modules/Z_edits/antag_edits/changeling_spiders.dm [hunk 2]
```diff
@@ -47,5 +47,5 @@
 	//We created a spider, let us announce this to ourselves.
 	ling_datum.spiders_created++
 
-//	new /obj/effect/mob_spawn/spider/bloody(user.loc) // Handled by Parent, so call ..() instead
+	//Parent handles new /obj/effect/mob_spawn/spider/bloody(user.loc)
```
Comment (JohnFulpWillard) [inline fulp_modules/Z_edits/antag_edits/changeling_spiders.dm:50]: I'm pretty sure this is meant to look like parent so it's easier to spot

Comment (Bilbo367) [reply fulp_modules/Z_edits/antag_edits/changeling_spiders.dm:50]: So you think the old comment is better?

Comment (JohnFulpWillard) [reply fulp_modules/Z_edits/antag_edits/changeling_spiders.dm:50]: Yea

**Comment by SgtHunk** [comment]
sounds/sound should probably just be sounds/
bit redundant otherwise

**Comment by SgtHunk** [comment]
sounds/sound should probably just be sounds/
bit redundant otherwise

File: fulp_modules/Z_edits/antag_edits/changeling_spiders.dm (hunk 2 above)
Comment (SgtHunk) [reply fulp_modules/Z_edits/antag_edits/changeling_spiders.dm:50]: Agree

File: fulp_modules/mapping/station_objects/decals.dm [hunk 3]
```diff
@@ -41,7 +41,7 @@
 	name = "prison fourcorner"
 
 /obj/effect/turf_decal/tile/security
-	color = "#FF0000"
+	color = COLOR_RED
```
Comment (SgtHunk) [inline fulp_modules/mapping/station_objects/decals.dm:44]: its fucked up we have these tiles...

**Review by SgtHunk — APPROVED** [review]

**Pull request merged**